cmd/autogen.sh

```diff
@@ -33,14 +33,14 @@ case "$ID" in
 		extra_opts="--libexecdir=/usr/lib/snapd"
 		;;
 	ubuntu)
-		case "$VERSION_ID" in
-			16.04)
-				extra_opts="--libexecdir=/usr/lib/snapd --enable-nvidia-multiarch --enable-static-libcap --enable-static-libapparmor --enable-static-libseccomp"
-				;;
-			*)
-				extra_opts="--libexecdir=/usr/lib/snapd --enable-nvidia-multiarch --enable-static-libcap"
-				;;
-		esac
+		extra_opts="--libexecdir=/usr/lib/snapd --enable-nvidia-multiarch --enable-static-libcap --enable-static-libapparmor --enable-static-libseccomp"
+		dpkg_architecture=$(which dpkg-architecture 2>/dev/null)
+		if [ -n "$dpkg_architecture" ]; then
```

Comment:
Is this test necessary? What's the point of checking if this is empty?

Reply:
Right, it's available in all releases we care about. Will update.